js/renderers/VerticalLayout.ts

```diff
@@ -26,9 +25,19 @@ class VerticalLayout implements jsonforms.services.IRenderer {
         var renderedElements = renderElements(element.elements);
 
         return {
-            "type": "VerticalLayout",
+            "type": "Layout",
             "elements": renderedElements,
-            "size": 99
+            "size": 99,
+            "template":
+                `<fieldset class="col-sm-{{element.size}}">
```

Comment:
Please remove the class from the fieldset. The size is already set on the parent div